src/eyepy/io/__init__.py

```diff
@@ -17,6 +18,33 @@
 logger = logging.getLogger("eyepy.io")
 
 
+def import_fda(path: Union[str, Path]) -> EyeVolume:
+    """ Read a Topcon fda file
+
+    This function is a wrapper around the FDA reader in OCT-Converter.
+
+    Args:
+        path: Path to the fda file
+
+    Returns:
+        Parsed data as EyeVolume object
+
+    """
+    reader = FDA(path)
+    oct_volume = reader.read_oct_volume()
```

Comment:
The FDA class has another function `read_oct_volume_2()`. Do you know when to use which function or whether it would make sense to automatically use this function in case `read_oct_volume()` fails?

Reply:
ok then. I added just in case.